app/views/shared/check_answers/_item.html.erb

```diff
@@ -9,7 +9,21 @@
   </dt>
   <!--  in-line styling due to issues around pdf generation in our production envs -->
```

Comment:
Just wondering if this same issue is going to affect the 'no_border' rendering properly above? As this used to probably be done in the same way as 'no_border' is now? Not sure though